src/components/events/AdminEvents/AdminEvents.jsx

```diff
@@ -3,8 +3,8 @@ import { Link } from 'react-router-dom';
 import { Input, Button, Radio, Row, Col, Card, Typography } from 'antd';
 import { FilterOutlined, PlusOutlined, SearchOutlined } from '@ant-design/icons';
 import { AFCBackend } from '../../../util/utils';
-import EventCard from '../event/EventCard/EventCard';
-import EventList from '../event/EventList/EventList';
+import EventCard from '../Event/EventCard/EventCard';
+import EventList from '../Event/EventList/EventList';
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.26.0/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "../Event/EventList/EventList"